app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  def likes_counter
+    post.increament!(:LikesCounter)
+  end
```

Comment:
- [ ] Kindly make use of the model callback functions such as `after_save` so that the counters are triggered every time a new like is saved.

```ruby
  after_save :update_likes_counter

  private

  def update_likes_counter
    post.increment!(:likes_counter)
  end
```


![counters](https://github.com/derrick1451/Blog-app/assets/53124548/1dd768cb-0655-4e4f-9015-6f1f77aa42a0)